### PR TITLE
Updated Results and Added a Legend

### PR DESCRIPTION
**Pull request by dhruv0000**
**Description**
Updated SMI result data and added Legend to the table.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Landscape-grid/smi.js [hunk 1]
```diff
@@ -40,19 +40,10 @@ function SMI_Compatibility() {
   return (
     <Tabs style={{ overflow: "auto", whiteSpace: "nowrap"}} className="landscape-table">
       <TabList>
-        <Tab>V0.5.0</Tab>
-        <Tab>V0.6.0</Tab>
-        <Tab>V0.4.0</Tab>
+        <Tab>V0.1.0</Tab>
```
Comment (leecalcote) [inline src/sections/Landscape-grid/smi.js:43]: Is this SMI's version number or the version number of our test suite? @dhruv0000 @kumarabd

Comment (kumarabd) [reply src/sections/Landscape-grid/smi.js:43]: Test suite

Comment (leecalcote) [reply src/sections/Landscape-grid/smi.js:43]: Thank you, sir.

Comment (leecalcote) [reply src/sections/Landscape-grid/smi.js:43]: Proposal: let's have the test suite version numbers follow/be that of Learn Layer5's version numbers - https://github.com/layer5io/learn-layer5/releases. What do you think?

**Comment by Nikhil-Ladha** [comment]
There are few conflicts @dhruv0000....

**Review by leecalcote — APPROVED** [review]
LGTM

File: src/sections/Landscape-grid/smi.js (hunk 1 above)
Comment (dhruv0000) [reply src/sections/Landscape-grid/smi.js]: Would have made sense if it was an independent repo. How will changes not related to smi-tests affect the table?

Comment (leecalcote) [reply src/sections/Landscape-grid/smi.js]: You're right - those changes may not affect the table. Other changes to the test criteria may or may not affect the table either. Changes to either the test criteria or the test tool (and it's behavior) warrant a bump in version number as changes to the tool or changes to the criteria can both affect test results.

**Pull request merged**